tools/justfile

```
@@ -0,0 +1,144 @@
+# Enable positional args
+set positional-arguments
+
+# Default values for the deployment
+tag    := "test"
+queue  := "active"
+worker := "htcmock"
+
+# Base compose file
+export COMPOSE_BASE := "../docker-compose/docker-compose.yml"
+
+# loggin level of deployment
+export LOGGING_LEVEL := "Information"
+
+# Sets the queue
+export QUEUE := if queue == "rabbit" {
+  "../docker-compose/docker-compose.queue-rabbitmq.yml"
+}else if queue == "rabbit091"{
+  "../docker-compose/docker-compose.queue-rabbitmq.yml"
+}else if queue == "artemis"{
```

Could you add whitespaces?
```suggestion
} else if queue == "artemis" {
```